### ISSUE TITLE
React component not properly exported after Gatsby 2.30 upgrade

### ISSUE THREAD

**Issue by openscript** [zabute/gatsby-plugin-svgr#47]
I ran into the following error:

> Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

This error originates from a SVG which I use the following way:

```
import { ReactComponent as WelcomeGraphic } from '../../static/graphics/welcome.svg';

<WelcomeSpotlight color="--white-color" background={<WelcomeGraphic />}>
```

I'm using this plugin with the following `.svgo.yml` configuration:

```
multipass: true
plugins:
  - removeDimensions: true
  - removeViewBox: false
```

I also have some types declared for it as I'm using TypeScript:

```
declare module '*.svg' {
  import { PureComponent, SVGProps } from 'react';

  export class ReactComponent extends PureComponent<SVGProps<SVGSVGElement>> {}
}
```

Do you have an idea what needs to be changed to have a proper default export back?

Thank you in advance.

**Comment by nonAlgebraic** [comment]
Same here. I believe this is caused because of a change to the existing webpack loader rule in Gatsby v2.30.x, which changed the `test` regex from `\.(ico|svg|jpg|jpeg|png|gif|webp)(\?.*)?$` to `\.(ico|svg|jpg|jpeg|png|gif|webp|avif)(\?.*)?$`, so this plugin's attempt to find that rule in the webpack config by searching for the former string fails, and the new rule is never added.

**Comment by coreyward** [comment]
Fixed in #48 and released in v2.1.0 with backward compatibility.
